src/pages/Profile/Sidebar/_Sidebar.scss

```diff
@@ -153,6 +153,21 @@
   display: flex;
   flex-direction: column;
   align-items: center;
+  background: rgb(248, 248, 252);
+  .gravatar-info-container {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    max-width: 500px;
+    margin-top: 10px;
+    margin-bottom: 10px;
+    border-bottom: 0.5px solid rgb(200, 200, 200);
+    padding: 10px 40px 20px 40px;
+    color: rgb(74, 89, 90);
+    .gravatar-info-main {
```

Comment:
looks like we're not using this class so we could remove or condense to `.gravatar-info-main {}`.